Source/com/drew/metadata/MetadataContext.java

```diff
@@ -0,0 +1,49 @@
+package com.drew.metadata;
+
+import com.drew.lang.annotations.NotNull;
+
+import java.util.Locale;
+
+/**
+ * Context class for metadata. Contains settings used while extracting and formatting metadata.
+ *
+ * @author Roel van Dijk
+ */
+public class MetadataContext
+{
+    /**
+     * Locale used to format metadata.
+     */
+    private Locale _locale;
```

Comment:
I would make `MetadataContext` immutable so that it can be passed around without worrying about threads. Thus, this, and any other fields in this class should be `final` in my opinion.

Reply:
I fully agree with that! Since we only have a `Locale` for now, I could create a constructor on `MetadataContext`.

However, using the builder pattern would make this more future proof (albeit more verbose).